File: Python Programming/problems on bitwise operator/problem13.py
'''
13. Write a program which accept one number and position from user and on that bit. Return modified number. 
	
	  Input :  10 4
	  Output :  14 
	
	Author : Milind Nalawade
	
	Date : 03/08/2021
	
	Language : Python 3.9.0

'''
import sys

#====
#Logic
#====
def ChkBit(no,pos):
	flag=0
	flag=no|pos
	return flag;
		
#=============
#Entry point function
#=============
def main():
	no=int(sys.argv[1])
	pos=int(sys.argv[2])
	ret=int(ChkBit(no,pos))
	print("\n Output : {} \n".format(ret));	
#=====
#starter
#=====	
if __name__=="__main__":
	main()

File: C Programming/problems on bitwise operator/problem13.c
/*
13. Write a program which accept one number and position from user and on that bit. Return modified number. 
	
	  Input :  10 4
	  Output :  14 
	
	Author : Milind Nalawade
	
	Date : 03/08/2021
	
	Language : C programming
	
*/
#include<stdio.h>
#include<stdlib.h>

//Logic 
int ChkBit(int no,int pos)
{
	int flag=0;
	flag=no|pos;
	return flag;		
}

//Entry point function 
int main(int argc,char * argv[])
{
	int no=atoi(argv[1]);
	int pos=atoi(argv[2]);
	int ret=ChkBit(no,pos);
	printf("\n Output : %d \n",ret);
	return 0;
}		